src/components/menu-top/ImportCalChart3.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <div class="modal-card">
+    <header class="modal-card-head">
+      <p class="modal-card-title">
+        Import Calchart3
+      </p>
+    </header>
+
+    <section class="modal-card-body">
+      <b-field label="Show title"/>
+      <b-field class="file">
+        <b-upload v-model="file" accept='.shw' @input="$parent.close()">
+          <a class="button is-primary">
+            <b-icon icon="upload"></b-icon>
+            <span>Click to upload</span>
+          </a>
+        </b-upload>
+        <span class="file-name" v-if="file">
+            {{ file.name }}
+        </span>
+      </b-field>
+
+    </section>
+
+    <footer class="modal-card-foot">
+      <b-button
+        type="is-primary"
+        data-test="file-modal--close"
+        @click="$parent.close()"
+      >
+        Close
+      </b-button>
+    </footer>
+  </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import LoadShow from '@/models/util/ParseCalChart3File';
+
+export default Vue.extend({
+  name: 'ImportCalChart3',
```

Comment:
I feel like we can make this generic so it can accept both Calchart 3 and 4 files. We can differentiate between the two versions on import. If the file is a JSON object, then we know it is Calchart 4. We can worry about differentiating later in a different issue, but for now, we can rename this to be a generic import show file modal.
```suggestion
  name: 'ImportShow',
```

Reply:
That's a great idea.  I'll make the change

Reply:
👍